vllm/model_executor/models/bert_embedding.py

```diff
@@ -0,0 +1,399 @@
+            # Skip the specific downstream task weight.
+            if name.startswith('cls.'):
+                continue
+
+            name = name[len(_prefix):] if name.startswith(_prefix) else name
+            name = _fix_key(name)
+
+            # use Pooler instead.
+            if name.startswith('pooler.'):
+                continue
+
+            for (param_name, weight_name, shard_id) in stacked_params_mapping:
+                if weight_name not in name:
+                    continue
+                name = name.replace(weight_name, param_name)
+
+                # Skip loading extra bias for GPTQ models.
+                if name.endswith(".bias") and name not in params_dict:
+                    continue
+
+                param = params_dict[name]
+                weight_loader = param.weight_loader
+                weight_loader(param, loaded_weight, shard_id)
+                break
+            else:
+                # Skip loading extra bias for GPTQ models.
+                if name.endswith(".bias") and name not in params_dict:
+                    continue
+
+                param = params_dict[name]
+                weight_loader = getattr(param, "weight_loader",
+                                        default_weight_loader)
+                weight_loader(param, loaded_weight)
+
+
+class BertModel(nn.Module):
+
+    def __init__(
+        self,
+        config: BertConfig,
+        cache_config: Optional[CacheConfig] = None,
+        quant_config: Optional[QuantizationConfig] = None,
+    ):
+        super().__init__()
+        self.embeddings = BertEmbedding(config)
+        self.encoder = BertEncoder(config, cache_config, quant_config)
+
+    def forward(
+        self,
+        input_ids: torch.Tensor,
+        position_ids: torch.Tensor,
+        kv_caches: List[torch.Tensor],
+        attn_metadata: AttentionMetadata,
+        inputs_embeds: Optional[torch.Tensor] = None,
+    ) -> torch.Tensor:
+        hidden_states = self.embeddings(input_ids=input_ids,
+                                        position_ids=position_ids,
+                                        inputs_embeds=inputs_embeds)
+        output = self.encoder(hidden_states, kv_caches, attn_metadata)
+        return output
+
+
+class BertEmbedding(nn.Module):
+
+    def __init__(self, config: BertConfig):
+        super().__init__()
+        self.size = config.hidden_size
+
+        self.word_embeddings = nn.Embedding(config.vocab_size,
+                                            config.hidden_size,
+                                            padding_idx=config.pad_token_id)
+        self.position_embeddings = nn.Embedding(config.max_position_embeddings,
+                                                config.hidden_size)
+        self.token_type_embeddings = nn.Embedding(config.type_vocab_size,
+                                                  config.hidden_size)
+        self.LayerNorm = nn.LayerNorm(config.hidden_size,
```

Comment:
Follow naming standards for layernorm
```suggestion
        self.layernorm = nn.LayerNorm(config.hidden_size,
```

Reply:
Followed the suggestion, and added renaming of LayerNorm parameter when loading weights.